vyxal/elements.py

```diff
@@ -416,12 +416,12 @@ def boolify(lhs, ctx):
     (any) -> is truthy?
     """
     if vy_type(lhs, simple=True) is list:
```

Comment:
Could you make this `if ctx.vectorise_boolify and vy_type(lhs, simple=True) is list:` instead? That way, the if would be simply `vectorise(boolify, lhs, ctx=ctx)` instead of a nested `if`.